api/tests/test_reservation_expe.py

```diff
@@ -218,7 +219,7 @@ def test_updates_bad_data(self):
 
         response = self.client.put(
```

Comment:
Ah here put is used instead of patch as well... maybe these tests could be updated too if we stick with patch for consistency

Reply:
Definitely

Reply:
Solved here : https://github.com/betagouv/ma-cantine/pull/1237/commits/40feb3d6e217efb51e6c435e0ee8010c353bef67